tests/blockchain/test_blockchain.py

```diff
@@ -460,90 +461,95 @@ async def test_genesis_no_icc(self, empty_blockchain):
 
     @pytest.mark.asyncio
     async def test_invalid_icc_sub_slot_vdf(self):
```

Comment:
You could avoid this indent by saying
```suggestion
    async def test_invalid_icc_sub_slot_vdf(self):
        ... old code here ...
        
    async def test_invalid_icc_sub_slot_vdf(self, keychain):
        with TempKeyring() as keychain:
            self.do_invalid_icc_sub_slot_vdf(keychain)
```